oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+account-analytic https://github.com/arkostyuk/account-analytic 11.0-mig-analytic_base_department
```

Comment:
@arkostyuk drop this

Reply:
Don't we want to keep at least `account-analytic` or travis will fail?